xla/service/gpu/ir_emission_utils.h

```diff
@@ -23,6 +23,7 @@ limitations under the License.
 #include <variant>
 #include <vector>
 
+#include "absl/container/flat_hash_map.h"
```

Comment:
This needs a BUILD file dependency

Reply:
Oops, flat_hash_set != flat_hash_map. Sorry about that.